styles/web/_vars.scss

```diff
@@ -750,1222 +250,975 @@ $sprk-btn-tertiary-border: $sprk-btn-border-width $sprk-btn-border-style
+/// The left position of the after element of Light Links on hover.
+$sprk-link-light-hover-after-left: 0% !default;
+/// The width of the after element of Light Links on hover.
+$sprk-link-light-hover-after-width: 100% !default;
+/// The underline style of Light Links on hover.
+$sprk-link-light-hover-border-bottom: $sprk-link-light-hover-underline-width
+  $sprk-link-border-bottom-style $sprk-link-light-hover-underline-color !default;
+/// The underline color value of Light Links.
+$sprk-link-light-underline-color: transparent !default;
+/// The underline width value of Light Links.
+$sprk-link-light-underline-width: 0.125rem !default;
+/// The font weight of Light Links.
+$sprk-link-light-font-weight: $sprk-font-weight-body-two !default;
+/// The underline style of Light Links.
+$sprk-link-light-border-bottom: $sprk-link-light-underline-width
+  $sprk-link-border-bottom-style $sprk-link-light-underline-color !default;
+/// The underline style of visited Light Links.
+$sprk-link-light-visited-border-bottom: $sprk-link-light-underline-width
+  $sprk-link-border-bottom-style $sprk-link-light-underline-color !default;
+/// The border settings for Plain Links.
+$sprk-link-plain-border: none !default;
+/// The border settings for Plain Links in the visited state.
+$sprk-link-plain-visited-border: none !default;
+/// The color of icons in Links containing icons.
+$sprk-link-has-icon-color-icon: $sprk-purple !default;
+/// The fill color for icons contained in Links.
+$sprk-link-has-icon-fill: $sprk-link-has-icon-color-icon !default;
+/// The stroke color for icons contained in Links.
+$sprk-link-has-icon-stroke: $sprk-link-has-icon-color-icon !default;
+/// The underline color of Links containing icons.
+$sprk-link-has-icon-underline-color: transparent !default;
+/// The underline width of Links containing icons.
+$sprk-link-has-icon-underline-width: 0 !default;
+/// The font weight of Links containing icons.
+$sprk-link-has-icon-font-weight: $sprk-font-weight-body-one !default;
+/// The border settings for Links with icons.
+$sprk-link-has-icon-border-bottom: $sprk-link-has-icon-underline-width
+  $sprk-link-border-bottom-style $sprk-link-has-icon-underline-color !default;
+/// The color value for Links with icons in the hover state.
+$sprk-link-has-icon-hover-color-text: $sprk-purple-deep !default;
+/// The fill color for icons contained in Links on hover.
+$sprk-link-has-icon-hover-fill: $sprk-link-has-icon-hover-color-text !default;
+/// The stroke color for icons contained in Links on hover.
+$sprk-link-has-icon-hover-stroke: $sprk-link-has-icon-hover-color-text !default;
+/// The border settings for Links with icons in the hover state.
+$sprk-link-has-icon-hover-border-bottom: $sprk-link-has-icon-underline-width
+  $sprk-link-border-bottom-style $sprk-link-has-icon-underline-color !default;
+/// The disabled Link
+$sprk-link-disabled-color: $sprk-black-tint-50 !default;
+/// The Disabled Link border-bottom
```

Comment:
Lets fix up this comment, add a period etc in the link.json file

Reply:
Got this!

Reply:
DONE! 
https://github.com/sparkdesignsystem/spark-design-system/pull/3663/files#diff-d2347dabbd8190c536ae2be298ed44f894d299f49372146541bbedcd30a16bcc